it/pom.xml

```diff
@@ -34,6 +34,31 @@
     <dataflow-api.version>v1b3-rev20220920-2.0.0</dataflow-api.version>
   </properties>
 
+  <!-- TODO: remove this block once libraries-bom gets past 26.19.0 -->
```

Comment:
Good news is that Beam 2.50.0 was updated to 26.21.0 https://github.com/apache/beam/pull/27780, so this is ~3-4 weeks from now

Reply:
ack!